### DISCUSSION TITLE
Would be nice to have an option to open menu links in a new tab instead of iframe

### DISCUSSION THREAD

**Discussion by fabfurnari** [VA3HDL/hamdashboard#32] [Ideas]
Some websites doesn't allow embedding in iframe, as you pointed out in the README this could be sometimes avoided with on-the-fly editing of the X-Frame-Options but it's a bit complex to achieve and doesn't work every time.

Another option could be specify some sort of parameter in the menu links array that will open the website in a different tab. This way the menu could become also a sort of "bookmark manager" for important websites that doesn't allow embedding (for example I would add https://hf.dxview.org/) 

Thanks!

**Comment by VA3HDL** [comment]
Hi Frabizio,

Will add to the To Do list. Thanks for the suggestion.
